src/Components/Form/InputText.jsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './form-scss/form-component.scss';
+
+const InputText = (props) => {
+  // typechecking with propTypes
+  InputText.propTypes = {
```

Comment:
An incorrect place to add prop validation check https://reactjs.org/docs/typechecking-with-proptypes.html

Reply:
also add prop type for all variables